test/ICSharpCode.SharpZipLib.Tests/TestSupport/RingBuffer.cs

```diff
@@ -21,14 +21,15 @@ public class ReadWriteRingBuffer
 		/// Create a new RingBuffer with a specified size.
 		/// </summary>
 		/// <param name="size">The size of the ring buffer to create.</param>
-		public ReadWriteRingBuffer(int size)
+		public ReadWriteRingBuffer(int size, CancellationToken? token = null)
```

Comment:
CancellationToken was added to fully support threaded testing without Thread.Abort